terraform/vars.tf

```diff
@@ -0,0 +1,90 @@
+variable "region" {
+  description = "The AWS region to create resources in."
+  default = "us-east-1"
+}
+
+variable "remote_state_backend_type" {
+  description = “The remote state backend, if you’re planning to hook this up to an existing DevSecOps-Infrastructure deployment.”
+  type = "string"
+  default = “s3”
```

Comment:
This, and other files, are full of "smart quotes". Notice the visual difference in quotes between the `type` and `default` values. You'll probably want to clean this up and use standard quoting.

Reply:
That's what I get for using an iPad editor.